Gruntfile.js

```diff
@@ -172,20 +169,10 @@ module.exports = function (grunt) {
         }
     };
 
-    gruntConfig.csslint = {
```

Comment:
Why are we removing these linter configs?
I think thay were probably added for consistency in CSS too!

Reply:
CSSLint is not maintained anymore
Here is the related changes: https://github.com/yabwe/medium-editor/issues/1503#issuecomment-552683079